src/main/java/frc/robot/subsystems/Claw.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+public class Claw extends SubsystemBase {
+  private DoubleSolenoid gripSolenoid;
+  private WPI_TalonFX leftClawMotor;
+  private WPI_TalonFX rightClawMotor;
+
+  public Claw() {
+    gripSolenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, 0, 1);
+    leftClawMotor = new WPI_TalonFX(0);
+    rightClawMotor = new WPI_TalonFX(1);
+  }
+
+  public void setGrip(DoubleSolenoid.Value position) {
+    gripSolenoid.set(position);
+  }
```

Comment:
Something you could change if you want is split this into two methods. closeGrip() and openGrip(). It would make it slightly easier for anyone writing commands to not have to worry about what it means when the solenoid is in each position.